entities/Rules/index.js

```diff
@@ -476,6 +470,27 @@ module.exports = {
       }
     },
   ],
+  LIQUIDATION: [
+    ({ UI, notify }, gameState) => {
+      const liquidateOption = require('../PlayerActions').prompt(
+        { notify, UI },
+        gameState,
+        // TODO: be smart about what options are available to the user?
+        ['MANAGE_PROPERTIES', 'TRADE', 'BANKRUPTCY', 'CANCEL']
```

Comment:
If the `CANCEL` is going to be a pattern, we should consider pulling that out

Reply:
Yes, we plan to change this into a pattern in the future. Especially with the `TODO` here.

Reply:
#46